inst/touchstone-receive.yaml

```diff
@@ -4,16 +4,23 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref }}
   cancel-in-progress: true
 
-on: pull_request
+on: 
+  pull_request:
+  #issue_comment:
+  #  types: ["created", "edited"]
 
 jobs:
   prepare:
     runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' || (
```

Comment:
is everyone eligible to trigger these runs? Maybe this should be more limited to collaborators.

Reply:
Sure, maybe also as an option in `use_touchstone`?